src/_locales/ja/messages.json

```diff
@@ -258,5 +259,29 @@
     "wallet_notification": "これはデフォルトの仮想通貨ウォレット以外の新しい仮想通貨ウォレットを作成します。仮想通貨ウォレットの秘密鍵を安全に保管する責任はあなたにあります。また、Maskbook は Ethereum 上での取引の支払いを行うことができませんのでご注意ください。",
     "wallets": "仮想通貨ウォレット",
     "untitled_wallet": "無題の財布",
-    "your_message": "あなたのメッセージ..."
-}
\ No newline at end of file
+    "your_message": "あなたのメッセージ...",
+    "plugin_file_service_display_name": "Plugin: File Service",
```

Comment:
I think you should leave it empty then fallback to en string in the runtime so it can up to date with the en translation